app/assets/javascripts/discourse/lib/show-modal.js.es6

```diff
@@ -12,7 +12,7 @@ export default function(name, opts) {
   //
   // Also, very little value in showing main outlet and header on iOS
   // anyway, so just hide it.
-  if (isAppleDevice()) {
+  if (opts.checkIfAppleDevice && isAppleDevice()) {
```

Comment:
I am confused then, do we need to even keep this "isAppleDevice" check anymore? Why not simply drop it?

Reply:
No need to keep it, I just wasn't sure how long we need to carry this kind of fixes for old iOS versions.